hbase-client/src/main/java/org/apache/hadoop/hbase/client/OnlineLogRecord.java

```diff
@@ -53,6 +56,18 @@ final public class OnlineLogRecord extends LogEntry {
         if (slowLogPayload.getMultiServiceCalls() == 0) {
           jsonObj.remove("multiServiceCalls");
         }
+        if (slowLogPayload.getRequestAttributes().isEmpty()) {
+          jsonObj.remove("requestAttributes");
+        } else {
+          jsonObj.add("requestAttributes",
+            gson.toJsonTree(deserializeAttributes(slowLogPayload.getRequestAttributes())));
+        }
+        if (slowLogPayload.getConnectionAttributes().isEmpty()) {
+          jsonObj.remove("connectionAttributes");
+        } else {
+          jsonObj.add("connectionAttributes",
+            gson.toJsonTree(deserializeAttributes(slowLogPayload.getConnectionAttributes())));
+        }
```

Comment:
If present, this will spit out a blob like:
```json
{
  "startTime": 1,
  "processingTime": 2,
  "queueTime": 3,
  "responseSize": 4,
  "blockBytesScanned": 5,
  "multiGetsCount": 6,
  "multiMutationsCount": 7,
  "requestAttributes": {
    "r": "1"
  },
  "connectionAttributes": {
    "c": "1"
  }
}
```